cmd/kubeadm/app/apis/kubeadm/v1alpha1/types.go

```diff
@@ -76,6 +81,8 @@ type Etcd struct {
 	KeyFile   string            `json:"keyFile"`
 	DataDir   string            `json:"dataDir"`
 	ExtraArgs map[string]string `json:"extraArgs"`
+	// Image specifies which container image to use for running etcd
+	Image string `json:"image"`
```

Comment:
We can specify that in the comment.  We should move to put way more comments into this file so it is self documenting and we can eventually perhaps generated docs.